### PR TITLE
Remove dead BitcoinUnits::id code and update assets-attribution for non-image unit selector

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

**Comment by Diapolo** [comment]
Tested ACK

**Comment by laanwj** [comment]
Trivial ACK

File: doc/assets-attribution.md [hunk 1]
```diff
@@ -39,9 +39,7 @@ Jonas Schnelli
     src/qt/res/icons/connect[0-3].png, src/qt/res/icons/eye_minus.png,
     src/qt/res/icons/eye_plus.png, src/qt/res/icons/verify.png,
     src/qt/res/icons/tx_inout.png, src/qt/res/icons/tx_input.png,
```
Comment (fanquake) [inline doc/assets-attribution.md]: care to fix the typo here as well? Save another commit s/tx_inout/tx_input

**Comment by fanquake** [comment]
Tested ACK

File: doc/assets-attribution.md (hunk 1 above)
Comment (luke-jr) [inline doc/assets-attribution.md]: ? there's 2 files, both listed

Comment (fanquake) [inline doc/assets-attribution.md]: eh, misread by me.

**Comment by jonasschnelli** [comment]
ACK

File: doc/assets-attribution.md (hunk 1 above)
Comment (laanwj) [inline doc/assets-attribution.md]: inout is correct in this case

**Comment by jtimon** [comment]
ACK

File: doc/assets-attribution.md [hunk 2]
```diff
@@ -39,9 +39,7 @@ Jonas Schnelli
...
-    src/qt/res/icons/unit_btc.png, src/qt/res/icons/unit_mbtc.png,
-    src/qt/res/icons/unit_ubtc.png, src/qt/res/src/verify.svg
-    src/qt/res/src/bitcoin.svg, src/qt/res/src/clock*.svg,
-    src/qt/res/src/connect*.svg, src/qt/res/src/mine.svg,
-    src/qt/res/src/qt.svg, src/qt/res/src/tx*.svg,
+    src/qt/res/src/verify.svg, src/qt/res/src/bitcoin.svg,
+    src/qt/res/src/clock*.svg, src/qt/res/src/connect*.svg,
+    src/qt/res/src/mine.svg, src/qt/res/src/qt.svg, src/qt/res/src/tx*.svg,
```
Comment (jonasschnelli) [inline doc/assets-attribution.md]: nit: normally we have two entries per line.

**Pull request merged**